ci/test.sh

```diff
@@ -47,12 +48,15 @@ traceStart "Testing Unity: Playmode :joystick:"
     pushd "workers/unity"
         dotnet run -p "${PROJECT_DIR}/.shared-ci/tools/RunUnity/RunUnity.csproj" -- \
             -batchmode \
+            -nographics \
             -enableCodeCoverage \
             -coverageResultsPath "${COVERAGE_RESULTS_PATH}/playground" \
             -coverageOptions "${COVERAGE_OPTIONS}" \
             -projectPath "${PROJECT_DIR}/workers/unity" \
             -runTests \
-            -testPlatform playmode \
+            -testPlatform "StandaloneWindows64" \
+            -buildTarget "StandaloneWindows64" \
```

Comment:
Should  this run on the standalone mac client if this is on a MacOS agent?

Reply:
Good catch, yes it should